app/wasm/test/custom_msg_test.go

```diff
@@ -70,13 +70,14 @@ func TestMintMsg(t *testing.T) {
 	}}
 	err := executeCustom(t, ctx, osmosis, reflect, lucky, msg, sdk.Coin{})
 	require.NoError(t, err)
+	sunDenom := fmt.Sprintf("factory/%s/%s", lucky.String(), msg.CreateDenom.Subdenom)
```

Comment:
This is code smell that this isn't trivial to get.

Need to see how it looks inside the cosmwasm.